backend/python/app/graphql/types/entity_type.py

```diff
@@ -0,0 +1,35 @@
+from ...models import Entity as EntityModel
+from ...resources.entity_dto import EntityDTO
+
+import graphene
+from graphene import relay
+from graphene_sqlalchemy import SQLAlchemyObjectType
+
+class Entity(SQLAlchemyObjectType):
+    class Meta:
+        model = EntityModel
+        interfaces = (relay.Node, )
+
+class EntityInput(graphene.InputObjectType):
+    string_field = graphene.String(required=True)
+    int_field = graphene.Int(required=True)
+    string_array_field = graphene.List(graphene.String, required=True)
+    enum_field = graphene.String(required=True) # TODO: make stricter
+    bool_field = graphene.Boolean(required=True)
+
+def entity_input_to_dto_and_model(entity_data):
```

Comment:
Please roast this pattern, I have no clue what I'm doing. 
I convert entity_data to entity_model as the mutation throws a fit if the returned object lacks an `id` field. The typescript graphql configuration of the starter-code has `EntityRequestDTO` and `EntityResponseDTO`, which is a potential solution, though `Entity` (model) already exists. LMK what you think.